ui/src/notification.ts

```diff
@@ -14,7 +14,10 @@ export enum Level {
 interface Notification {
   id: ID;
   level: Level;
+  showIcon: boolean;
   message: string;
+  actionText: string;
```

Comment:
Should `actionText` and `actionHandler` be paired as some kind of type `Action { message: string, action: Handler }`.